### PR TITLE
VZ-6610 VZ-6611: Automated tests for Verrazzano release distribution bundles

### PR DESCRIPTION
**Pull request by adiforluls**
Implements VZ-6610 and VZ-6611

### CODE REVIEW

File: ci/verify-distribution/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,792 @@
...
+                        echo "Upload OCIR region: $ocirRegion"
+                        echo "Upload OCIR Repository: ${ocirRegistry}"
+                    """
+                    if (params.DISTRIBUTION_VARIANT == "Commercial") {
+                        sh """
+                            cd ${TARBALL_DIR}
+                            chmod +x ${TARBALL_DIR}/bin/vz-registry-image-helper.sh
+                            ${TARBALL_DIR}/bin/vz-registry-image-helper.sh -t ${ocirRegistry} -l ./images -r ${baseImageRepo}
+                        """
+                    }
```
Comment (adiforluls) [inline ci/verify-distribution/Jenkinsfile:308]: Add else statement accordingly for `OpenSource` distribution variant

Comment (adiforluls) [inline ci/verify-distribution/Jenkinsfile:308]: Add else statement accordingly for OpenSource variant

File: ci/verify-distribution/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,792 @@
...
+        stage('Download and extract Tarball') {
+            steps {
+                script {
+                    downloadLocation = "${WORKSPACE}/${zipFileLoc}"
+                    baseFilename = sh (
+                        script: "basename ${zipFileLoc} .zip",
+                        returnStdout: true
+                        ).trim()
+                    // Commercial bundle doesn't have a tarfile and CheckSum, add logic based on distribution variant
+                    tarfileName = ""
```
Comment (adiforluls) [inline ci/verify-distribution/Jenkinsfile:262]: Commercial bundle doesn’t have tar files for cli, add logic accordingly

File: ci/verify-distribution/Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,792 @@
...
+// - at some point, we can either enable bucket replication or have the pipeline push it to more regions
+//def agentLabel = env.JOB_NAME.contains('master') ? "phxlarge" : "VM.Standard2.8"
+def agentLabel = "phxlarge"
+def ocirRegion = "phx"
+def ocirRegistry = "${ocirRegion}.ocir.io"
+def imageRepoSubPath=""
+def baseImageRepo=""
+def EFFECTIVE_DUMP_K8S_CLUSTER_ON_SUCCESS = false
+def commercialBundleLocation="pabhat%2Fvz-commercial-bundle-last-clean-periodic-test/verrazzano-1.4.0-commercial.zip"
+def openSourceBundleLocation=""
```
Comment (adiforluls) [inline ci/verify-distribution/Jenkinsfile:15]: Both `commercialBundleLocation` and `openSourceBundleLocation` will default to locations where master is storing these bundles. Set this to whatever path is being used for testing for now.

File: ci/verify-distribution/Jenkinsfile [hunk 4]
```diff
@@ -0,0 +1,808 @@
...
+// Pin to PHX for now for testing; tarball is located only in PHX at present, and takes 15+ mins to download to LHR at runtime
+// - at some point, we can either enable bucket replication or have the pipeline push it to more regions
+//def agentLabel = env.JOB_NAME.contains('master') ? "phxlarge" : "VM.Standard2.8"
+def agentLabel = "phxlarge"
+def ocirRegion = "phx"
+def ocirRegistry = "${ocirRegion}.ocir.io"
+def imageRepoSubPath=""
+def baseImageRepo=""
+def EFFECTIVE_DUMP_K8S_CLUSTER_ON_SUCCESS = false
+def releaseBundlePrefix="master-last-clean-periodic-test"
```
Comment (agv426) [inline ci/verify-distribution/Jenkinsfile:14]: Use the CLEAN_BRANCH_NAME pattern here

File: ci/verify-distribution/Jenkinsfile [hunk 5]
```diff
@@ -0,0 +1,808 @@
...
+                    post {
+                        success {
+                            script {
+                                if (EFFECTIVE_DUMP_K8S_CLUSTER_ON_SUCCESS == true) {
+                                    dumpK8sCluster('install-success-cluster-snapshot')
+                                }
+                            }
+                        }
+                        always {
+                            archiveArtifacts artifacts: "acceptance-test-operator.yaml,downloaded-operator.yaml", allowEmptyArchive: true
```
Comment (bhatpmk) [inline ci/verify-distribution/Jenkinsfile:355]: This is not required, as we won't be having these files in WORKSPACE

**Review by agv426 — DISMISSED** [review]

File: ci/verify-distribution/Jenkinsfile (hunk 5 above)
Comment (saidu941) [reply ci/verify-distribution/Jenkinsfile]: Cleanedup and updated PR

**Review by agv426 — APPROVED** [review]

**Pull request merged**